apps/webcert/src/feature/certificate/CertificateSidePanel/CertificateSidePanel.tsx

```diff
@@ -30,14 +29,15 @@ const CertificateSidePanel: React.FC = () => {
     return link ? [...result, link] : result
   }, [])
 
+  const [selectedTabIndex, setSelectedTabIndex] = useState(0)
```

Comment:
Was this intensionally included in this PR?

Reply:
Yes